src/libs/LocaleDigitUtils.ts

```diff
@@ -8,12 +8,12 @@ const INDEX_DECIMAL = 10;
 const INDEX_MINUS_SIGN = 11;
 const INDEX_GROUP = 12;
 
-const getLocaleDigits = _.memoize((locale) => {
-    const localeDigits = _.clone(STANDARD_DIGITS);
+const getLocaleDigits = _.memoize((locale: string) => {
```

Comment:
```suggestion
const getLocaleDigits = _.memoize((locale: string): string[] => {
```

Let's make the function return type explicit!

Reply:
Done! @fabioh8010